docs/run.md

```diff
@@ -108,19 +108,19 @@ In that specific record, 3 transactions are executed:
 
 The first corresponds to the deployment of the lib `testLib`.
 
-The second corresponds to the deployment of the contract `test`, the
-first parameter of the constructor is set to 11. That contract depends
+The second corresponds to the deployment of the contract `test` with the
+first parameter of the constructor set to 11. That contract depends
 on a library. The linkage is done using the property `linkReferences`.
-In that case we use the addres of the previously created library :
+In that case we use the address of the previously created library :
 `created{1512830014773}`. the number is the id (timestamp) of the
```

Comment:
Please capitalize the T in the `the` of the new sentence - it has (incorrectly been):
**the** number is the id (timestamp) of the...